src/components/camera/CameraStream.js

```diff
@@ -55,13 +55,13 @@ function CameraStream({ cameraName }) {
           video: new Uint8Array(frameDataArray[i])
         });
       }
-      let time = document.getElementById(vidTag.props.id).currentTime;
-      setLastFrameTime(document.getElementById(vidTag.props.id).currentTime);
-      if (time !== lastFrameTime) {
+      const currentTime = Date.now();
+      if (currentTime !== lastFrameTime) {
         setCurrentFpsAvg((oldFps) => {
-          return (oldFps + (1 / ((time - lastFrameTime)))) / 2;
+          return (oldFps + 1 / ((currentTime - lastFrameTime) / 1000)) / 2;
         });
       }
+      setLastFrameTime(currentTime); // current time in ms
```

Comment:
Isn't this from your last pr? Make sure you're up to date with master so these extra changes don't show up